pnpm-lock.yaml

```diff
@@ -2710,6 +2710,9 @@ importers:
       '@types/react':
         specifier: 18.2.33
         version: 18.2.33
+      '@types/react-router-dom':
+        specifier: 5.3.3
+        version: 5.3.3
```

Comment:
I just did `pnpm install ....` and that's what it installed. I think @haqadn is best geared to answer this because I think he was actually the one converting our router.

Reply:
I have no idea why it installed this version. The `react-router-dom` version we are using is `6.21`. Maybe the types didn't change between these versions?